PF/PF-Assignment-1/Shopping.java

```diff
@@ -0,0 +1,132 @@
+package sc;
+import java.util.ArrayList;
+import java.util.List;
+public class Shopping {
+       
+    private List<Item> Cart = new ArrayList<Item>();
+    
+    ShoppingCart(){
+    	
+    }
+ 
+    public void addItem(String Pname,float Pprice,int Pquantity )
+    {
+        Item Product= new Item(Pname,Pprice,Pquantity);
+        Cart.add(Product);
+    }
+    
+    public void removeItem(String Pname) {
+       Item delprod = null;
```

Comment:
This variable is unused here. I think you added here to store the Item you are removing from the List. If that is so, then you should also return the deleted value instead of void.